message_broker_producer.install

```diff
@@ -68,8 +68,7 @@ function message_broker_producer_requirements($phase) {
 
   if ($phase == 'runtime') {
 
-    $phplib_library = libraries_detect('messagebroker-phplib');
-    $config_library = libraries_detect('messagebroker-config');
```

Comment:
The detection of `messagebroker-config` is important as well. Why are you removing it?